### PR TITLE
Fix wrong folder name

### PR DESCRIPTION
**Pull request by nitriques**
The proper folder name seems to be content, not data.

### CODE REVIEW

**Comment by khos2ow** [comment]
Nice catch! Thanks @nitriques :+1:

**Comment by khos2ow** [comment]
Using `yml` in `data` folder was the old way, and now it's changed to `md` file in `content` folder, can you adjust file format in your PR as well (`yml` -> `md`)?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ You can add a record using any of the following three methods. We have a few met
 
 - Complete PR
   - add record in [README](https://github.com/opensourcecities/montreal/blob/master/README.md) AND
-  - add record's `yml` file (based on [Type of Records](#type-of-records)) in `data` folder
```
Comment (khos2ow) [inline CONTRIBUTING.md]: Using `yml` in `data` folder was the old way, and now it's changed to `md` file in `content` folder, can you adjust file format in your PR as well (`yml` -> `md`)?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,10 +4,10 @@ You can add a record using any of the following three methods. We have a few met
...
+  - add record's `yml` file (based on [Type of Records](#type-of-records)) in `content` folder
```
Comment (khos2ow) [inline CONTRIBUTING.md]: Using `yml` in `data` folder was the old way, and now it's changed to `md` file in `content` folder, can you adjust file format in your PR as well (`yml` -> `md`)?

Comment (nitriques) [reply CONTRIBUTING.md]: @khos2ow Can you check d831737ab0 ? It is still yml !

Comment (khos2ow) [reply CONTRIBUTING.md]: That's fine. Your description was good. The base file is in `.md` but the metadata on top is in `yml`.

**Pull request merged**